### ISSUE TITLE
Easily create and maintain BigQuery tables

### ISSUE THREAD

**Issue by duizendstra** [gsuitedevs/solutions#60]
Solution title: Easily create and maintain BigQuery tables

One sentence summary: Create BigQuery tables with Google Sheets and Apps Script.

Description (1 - 2 paragraphs):
Creating BigQuery tables using the standard GUI can be a hassle. This app provides an alternative way to create BigQuery tables using the BigQuery Table builder sheet. This Google Spreadsheet automates the creation of the BigQuery tables, documents the tables and supports collaboration around the table schema design.

Design Details:
It is described in this article: https://medium.com/google-cloud/https-medium-com-duizendstra-create-bigquery-tables-with-google-sheets-and-apps-script-a0751b43a952

**Comment by asrivas** [comment]
This sounds like a great idea but I'd like to rephrase this to hide the implementation details from  the Solution & Summary. Our goal with the gallery is show solved example workflows/problems. Can you wrap this in a specific scenario? Some use case where the user needs to store the Sheet data elsewhere

Here is another solution that uses BQ (but only READ) where the title/description is focused on the tasks:
Generate custom sales reports automatically   
https://github.com/gsuitedevs/solutions/tree/master/generate-sales-report

**Comment by jpoehnelt** [comment]
Closing this issue as the repository is archived.
